test/keyboard-navigation.html

```diff
@@ -1416,20 +1420,22 @@
             right();
           });
 
-          it('should click first cell child on space', () => {
+          it('should click first cell child on space keyup', () => {
             const firstChild = getCellContent(header.children[0].children[2]).children[0];
             const clickStub = sinon.stub(firstChild, 'click');
 
-            space();
+            spaceDown();
+            spaceUp();
 
             expect(clickStub.called).to.be.true;
           });
```

Comment:
To verify that the bug is fixed, there should be also a test case for calling only `spaceDown()`, expecting the element not to be clicked.

Reply:
Good point! I have enhanced the existing test.

Reply:
I'm wondering - could this be the root issue of https://github.com/vaadin/flow/issues/6929 as well?

Reply:
Knoobie, good point, it looks related. But that issue seems to happen only with the Flow integration and component renderers, so probably not directly related to this.

Yuriy, tests ok :+1: